### ISSUE TITLE
HUDLoggerView dissapears after sharing the LogFile

### ISSUE THREAD

**Issue by sanarroya** [JohnEstropia/JEToolkit#2]
Hi, I was testing JEDebugging and notice that every time after i shared the log file via Mail the HUDLoggerView dissapears, I don't know if this is a bug, or is the way is suposed to behave, i think it shouldn't dissapear.

Thanks

**Comment by JohnEstropia** [comment]
@sanarroya I can't reproduce this from my side. I use this functionality a lot myself and have never seen this bug. Does it behave differently between different iOS versions?

**Comment by brightsider** [comment]
@JohnEstropia see video of this bug - https://dl.dropboxusercontent.com/u/61174294/bug.mov

**Comment by JohnEstropia** [comment]
@brightsider @sanarroya 
Thanks for the report! I've found the cause but it only seems to happen on the iOS 9 simulator where the Mail's view controller crashes with "MailCompositionService quit unexpectedly". 

We can tell the HUDLoggerView to look for a new window, but at this point your app already has a crashed window instance lurking around (I've confirmed that the Mail's controller left it's keyboard window) so I don't think there will be an elegant way to solve this.
